EcpBtBot/MainClass.py

```diff
@@ -1,39 +1,36 @@
 
+import logging
 import os
+from .Conversational_Integration import *
```

Comment:
not a good practice to import all.... About modules, first thing is, they should be snake_case, which also means they are lower case. Second either import an explicit object, or import the module and use it as prefix of all the object you get there. but do not import `*` because it does not help understand where the objects come from and might lead to naming conflicts.

Reply:
ok